quadrat/child-experimental-theme.json

```diff
@@ -129,7 +129,8 @@
 				"list": {
 					"padding": {
 						"left": "calc( 2 * var(--wp--custom--margin--horizontal) )"
-					}
+					},
+					"sub-list-style-type": "disc"
```

Comment:
I'm not sure I'm keen to mix a variable not used in blank-canvas-blocks into the mix here.  I think it would be confusing and suggest we limit the 'custom' variables to only extend what BCB is doing to ponyfill gutenberg.

My 'druthers would be to just set the li types to 'disc' rather than use a variable at all.

OR to integrate this into Blank Canvas Blocks.  (Though I prefer just ... setting it in quadrat's scss).